### PR TITLE
[#2307] license: Add left jars with Apache license

### LINKED ISSUE #2307: [Bug] LICENSE-binary is missing lots of jars and we don't know if they have problematic licenses
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-uniffle/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

https://github.com/apache/incubator-uniffle/blob/master/LICENSE-binary has lots of jars listed but the list is far from complete.
I got this list from the 0.9.2-rc1 bin tar.gz.

Could you also various Apache licensed jars in alphabetic order? In existing LICENSE-binary, there is a jetty dependency listed away from all the other jetty dependencies (1 example but there could be more).

```
rss-0.9.2-hadoop2.8 % find . -name "*.jar"      
./jars/coordinator/perfmark-api-0.26.0.jar
./jars/coordinator/annotations-4.1.1.4.jar
./jars/coordinator/grpc-stub-1.61.1.jar
./jars/coordinator/netty-codec-stomp-4.1.106.Final.jar
./jars/coordinator/netty-transport-native-kqueue-4.1.106.Final-osx-x86_64.jar
./jars/coordinator/netty-codec-smtp-4.1.106.Final.jar
./jars/coordinator/animal-sniffer-annotations-1.23.jar
./jars/coordinator/slf4j-api-1.7.36.jar
./jars/coordinator/jetty-server-9.3.24.v20180605.jar
./jars/coordinator/grpc-api-1.61.1.jar
./jars/coordinator/jetty-util-9.3.24.v20180605.jar
./jars/coordinator/rss-proto-0.9.2.jar
./jars/coordinator/RoaringBitmap-0.9.15.jar
./jars/coordinator/netty-all-4.1.106.Final.jar
./jars/coordinator/proto-google-common-protos-2.29.0.jar
./jars/coordinator/jetty-io-9.3.24.v20180605.jar
./jars/coordinator/javax.annotation-api-1.3.2.jar
./jars/coordinator/j2objc-annotations-2.8.jar
./jars/coordinator/snakeyaml-2.2.jar
./jars/coordinator/picocli-4.5.2.jar
./jars/coordinator/netty-handler-4.1.106.Final.jar
./jars/coordinator/activation-1.1.1.jar
./jars/coordinator/grpc-context-1.61.1.jar
./jars/coordinator/netty-codec-dns-4.1.106.Final.jar
./jars/coordinator/netty-codec-redis-4.1.106.Final.jar
./jars/coordinator/jackson-core-2.10.0.jar
./jars/coordinator/javassist-3.25.0-GA.jar
./jars/coordinator/netty-transport-native-epoll-4.1.106.Final-linux-riscv64.jar
./jars/coordinator/netty-common-4.1.106.Final.jar
./jars/coordinator/netty-codec-haproxy-4.1.106.Final.jar
./jars/coordinator/gson-2.10.1.jar
./jars/coordinator/netty-buffer-4.1.106.Final.jar
./jars/coordinator/netty-resolver-dns-native-macos-4.1.106.Final-osx-x86_64.jar
./jars/coordinator/rss-internal-client-0.9.2.jar
./jars/coordinator/javax.servlet-api-3.1.0.jar
./jars/coordinator/netty-codec-xml-4.1.106.Final.jar
./jars/coordinator/hbase-shaded-jersey-4.1.4.jar
./jars/coordinator/netty-transport-4.1.106.Final.jar
./jars/coordinator/netty-handler-ssl-ocsp-4.1.106.Final.jar
./jars/coordinator/netty-handler-proxy-4.1.106.Final.jar
./jars/coordinator/netty-codec-socks-4.1.106.Final.jar
./jars/coordinator/netty-transport-classes-epoll-4.1.106.Final.jar
./jars/coordinator/simpleclient_jetty-0.9.0.jar
./jars/coordinator/grpc-util-1.61.1.jar
./jars/coordinator/failureaccess-1.0.1.jar
./jars/coordinator/netty-codec-mqtt-4.1.106.Final.jar
./jars/coordinator/shims-0.9.15.jar
./jars/coordinator/simpleclient_common-0.9.0.jar
./jars/coordinator/netty-codec-http-4.1.106.Final.jar
./jars/coordinator/checker-qual-3.37.0.jar
./jars/coordinator/netty-transport-rxtx-4.1.106.Final.jar
./jars/coordinator/jaxb-api-2.3.0.jar
./jars/coordinator/jetty-security-9.3.24.v20180605.jar
./jars/coordinator/netty-codec-4.1.106.Final.jar
./jars/coordinator/netty-transport-classes-kqueue-4.1.106.Final.jar
./jars/coordinator/netty-transport-native-unix-common-4.1.106.Final.jar
./jars/coordinator/error_prone_annotations-2.23.0.jar
./jars/coordinator/simpleclient-0.9.0.jar
./jars/coordinator/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
./jars/coordinator/protobuf-java-3.25.1.jar
./jars/coordinator/grpc-protobuf-lite-1.61.1.jar
./jars/coordinator/jakarta.inject-2.6.1.jar
./jars/coordinator/grpc-protobuf-1.61.1.jar
./jars/coordinator/rss-common-0.9.2.jar
./jars/coordinator/netty-resolver-4.1.106.Final.jar
./jars/coordinator/netty-resolver-dns-4.1.106.Final.jar
./jars/coordinator/simpleclient_pushgateway-0.9.0.jar
./jars/coordinator/simpleclient_httpserver-0.9.0.jar
./jars/coordinator/jetty-proxy-9.3.24.v20180605.jar
./jars/coordinator/jetty-http-9.3.24.v20180605.jar
./jars/coordinator/coordinator-0.9.2.jar
./jars/coordinator/netty-transport-native-epoll-4.1.106.Final-linux-x86_64.jar
./jars/coordinator/grpc-core-1.61.1.jar
./jars/coordinator/log4j-api-2.23.0.jar
./jars/coordinator/netty-transport-sctp-4.1.106.Final.jar
./jars/coordinator/guava-32.1.3-jre.jar
./jars/coordinator/jakarta.annotation-api-1.3.5.jar
./jars/coordinator/jetty-client-9.3.24.v20180605.jar
./jars/coordinator/simpleclient_servlet-0.9.0.jar
./jars/coordinator/lz4-1.3.0.jar
./jars/coordinator/netty-codec-http2-4.1.106.Final.jar
./jars/coordinator/netty-codec-memcache-4.1.106.Final.jar
./jars/coordinator/netty-transport-udt-4.1.106.Final.jar
./jars/coordinator/simpleclient_hotspot-0.9.0.jar
./jars/coordinator/jsr305-3.0.2.jar
./jars/coordinator/commons-lang3-3.10.jar
./jars/coordinator/jetty-servlet-9.3.24.v20180605.jar
./jars/coordinator/netty-resolver-dns-classes-macos-4.1.106.Final.jar
./jars/coordinator/netty-transport-native-kqueue-4.1.106.Final-osx-aarch_64.jar
./jars/coordinator/jakarta.validation-api-2.0.2.jar
./jars/coordinator/netty-resolver-dns-native-macos-4.1.106.Final-osx-aarch_64.jar
./jars/coordinator/grpc-netty-shaded-1.61.1.jar
./jars/coordinator/jackson-annotations-2.10.0.jar
./jars/coordinator/log4j-core-2.23.0.jar
./jars/coordinator/jackson-databind-2.10.0.jar
./jars/coordinator/protobuf-java-util-3.25.1.jar
./jars/coordinator/netty-transport-native-epoll-4.1.106.Final-linux-aarch_64.jar
./jars/coordinator/log4j-slf4j-impl-2.23.0.jar
./jars/server/perfmark-api-0.26.0.jar
./jars/server/annotations-4.1.1.4.jar
./jars/server/grpc-stub-1.61.1.jar
./jars/server/netty-codec-stomp-4.1.106.Final.jar
./jars/server/netty-transport-native-kqueue-4.1.106.Final-osx-x86_64.jar
./jars/server/netty-codec-smtp-4.1.106.Final.jar
./jars/server/animal-sniffer-annotations-1.23.jar
./jars/server/slf4j-api-1.7.36.jar
./jars/server/jetty-server-9.3.24.v20180605.jar
./jars/server/grpc-api-1.61.1.jar
./jars/server/jetty-util-9.3.24.v20180605.jar
./jars/server/shuffle-server-0.9.2.jar
./jars/server/rss-proto-0.9.2.jar
./jars/server/RoaringBitmap-0.9.15.jar
./jars/server/netty-all-4.1.106.Final.jar
./jars/server/proto-google-common-protos-2.29.0.jar
./jars/server/jetty-io-9.3.24.v20180605.jar
./jars/server/javax.annotation-api-1.3.2.jar
./jars/server/j2objc-annotations-2.8.jar
./jars/server/picocli-4.5.2.jar
./jars/server/netty-handler-4.1.106.Final.jar
./jars/server/activation-1.1.1.jar
./jars/server/grpc-context-1.61.1.jar
./jars/server/netty-codec-dns-4.1.106.Final.jar
./jars/server/netty-codec-redis-4.1.106.Final.jar
./jars/server/jackson-core-2.10.0.jar
./jars/server/javassist-3.25.0-GA.jar
./jars/server/netty-transport-native-epoll-4.1.106.Final-linux-riscv64.jar
./jars/server/netty-common-4.1.106.Final.jar
./jars/server/netty-codec-haproxy-4.1.106.Final.jar
./jars/server/gson-2.10.1.jar
./jars/server/netty-buffer-4.1.106.Final.jar
./jars/server/netty-resolver-dns-native-macos-4.1.106.Final-osx-x86_64.jar
./jars/server/rss-internal-client-0.9.2.jar
./jars/server/javax.servlet-api-3.1.0.jar
./jars/server/netty-codec-xml-4.1.106.Final.jar
./jars/server/hbase-shaded-jersey-4.1.4.jar
./jars/server/netty-transport-4.1.106.Final.jar
./jars/server/netty-handler-ssl-ocsp-4.1.106.Final.jar
./jars/server/netty-handler-proxy-4.1.106.Final.jar
./jars/server/netty-codec-socks-4.1.106.Final.jar
./jars/server/netty-transport-classes-epoll-4.1.106.Final.jar
./jars/server/simpleclient_jetty-0.9.0.jar
./jars/server/grpc-util-1.61.1.jar
./jars/server/failureaccess-1.0.1.jar
./jars/server/netty-codec-mqtt-4.1.106.Final.jar
./jars/server/shims-0.9.15.jar
./jars/server/simpleclient_common-0.9.0.jar
./jars/server/netty-codec-http-4.1.106.Final.jar
./jars/server/checker-qual-3.37.0.jar
./jars/server/netty-transport-rxtx-4.1.106.Final.jar
./jars/server/jaxb-api-2.3.0.jar
./jars/server/jetty-security-9.3.24.v20180605.jar
./jars/server/netty-codec-4.1.106.Final.jar
./jars/server/netty-transport-classes-kqueue-4.1.106.Final.jar
./jars/server/netty-transport-native-unix-common-4.1.106.Final.jar
./jars/server/error_prone_annotations-2.23.0.jar
./jars/server/simpleclient-0.9.0.jar
./jars/server/shuffle-storage-0.9.2.jar
./jars/server/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
./jars/server/protobuf-java-3.25.1.jar
./jars/server/grpc-protobuf-lite-1.61.1.jar
./jars/server/jakarta.inject-2.6.1.jar
./jars/server/grpc-protobuf-1.61.1.jar
./jars/server/rss-common-0.9.2.jar
./jars/server/netty-resolver-4.1.106.Final.jar
./jars/server/netty-resolver-dns-4.1.106.Final.jar
./jars/server/simpleclient_pushgateway-0.9.0.jar
./jars/server/simpleclient_httpserver-0.9.0.jar
./jars/server/jetty-proxy-9.3.24.v20180605.jar
./jars/server/commons-collections-3.2.2.jar
./jars/server/jetty-http-9.3.24.v20180605.jar
./jars/server/netty-transport-native-epoll-4.1.106.Final-linux-x86_64.jar
./jars/server/grpc-core-1.61.1.jar
./jars/server/log4j-api-2.23.0.jar
./jars/server/netty-transport-sctp-4.1.106.Final.jar
./jars/server/guava-32.1.3-jre.jar
./jars/server/jakarta.annotation-api-1.3.5.jar
./jars/server/jetty-client-9.3.24.v20180605.jar
./jars/server/simpleclient_servlet-0.9.0.jar
./jars/server/lz4-1.3.0.jar
./jars/server/netty-codec-http2-4.1.106.Final.jar
./jars/server/netty-codec-memcache-4.1.106.Final.jar
./jars/server/netty-transport-udt-4.1.106.Final.jar
./jars/server/simpleclient_hotspot-0.9.0.jar
./jars/server/jsr305-3.0.2.jar
./jars/server/commons-lang3-3.10.jar
./jars/server/jetty-servlet-9.3.24.v20180605.jar
./jars/server/netty-resolver-dns-classes-macos-4.1.106.Final.jar
./jars/server/netty-transport-native-kqueue-4.1.106.Final-osx-aarch_64.jar
./jars/server/jakarta.validation-api-2.0.2.jar
./jars/server/netty-resolver-dns-native-macos-4.1.106.Final-osx-aarch_64.jar
./jars/server/grpc-netty-shaded-1.61.1.jar
./jars/server/jackson-annotations-2.10.0.jar
./jars/server/log4j-core-2.23.0.jar
./jars/server/jackson-databind-2.10.0.jar
./jars/server/protobuf-java-util-3.25.1.jar
./jars/server/netty-transport-native-epoll-4.1.106.Final-linux-aarch_64.jar
./jars/server/log4j-slf4j-impl-2.23.0.jar
./jars/cli/perfmark-api-0.26.0.jar
./jars/cli/annotations-4.1.1.4.jar
./jars/cli/grpc-stub-1.61.1.jar
./jars/cli/commons-logging-1.2.jar
./jars/cli/netty-codec-stomp-4.1.106.Final.jar
./jars/cli/netty-transport-native-kqueue-4.1.106.Final-osx-x86_64.jar
./jars/cli/netty-codec-smtp-4.1.106.Final.jar
./jars/cli/animal-sniffer-annotations-1.23.jar
./jars/cli/slf4j-api-1.7.36.jar
./jars/cli/jetty-server-9.3.24.v20180605.jar
./jars/cli/grpc-api-1.61.1.jar
./jars/cli/jetty-util-9.3.24.v20180605.jar
./jars/cli/cli-0.9.2.jar
./jars/cli/rss-proto-0.9.2.jar
./jars/cli/RoaringBitmap-0.9.15.jar
./jars/cli/netty-all-4.1.106.Final.jar
./jars/cli/proto-google-common-protos-2.29.0.jar
./jars/cli/jetty-io-9.3.24.v20180605.jar
./jars/cli/javax.annotation-api-1.3.2.jar
./jars/cli/j2objc-annotations-2.8.jar
./jars/cli/picocli-4.5.2.jar
./jars/cli/netty-handler-4.1.106.Final.jar
./jars/cli/commons-cli-1.5.0.jar
./jars/cli/activation-1.1.1.jar
./jars/cli/grpc-context-1.61.1.jar
./jars/cli/netty-codec-dns-4.1.106.Final.jar
./jars/cli/netty-codec-redis-4.1.106.Final.jar
./jars/cli/jackson-core-2.10.0.jar
./jars/cli/javassist-3.25.0-GA.jar
./jars/cli/netty-transport-native-epoll-4.1.106.Final-linux-riscv64.jar
./jars/cli/netty-common-4.1.106.Final.jar
./jars/cli/netty-codec-haproxy-4.1.106.Final.jar
./jars/cli/gson-2.10.1.jar
./jars/cli/netty-buffer-4.1.106.Final.jar
./jars/cli/netty-resolver-dns-native-macos-4.1.106.Final-osx-x86_64.jar
./jars/cli/rss-internal-client-0.9.2.jar
./jars/cli/javax.servlet-api-3.1.0.jar
./jars/cli/netty-codec-xml-4.1.106.Final.jar
./jars/cli/hbase-shaded-jersey-4.1.4.jar
./jars/cli/netty-transport-4.1.106.Final.jar
./jars/cli/netty-handler-ssl-ocsp-4.1.106.Final.jar
./jars/cli/netty-handler-proxy-4.1.106.Final.jar
./jars/cli/netty-codec-socks-4.1.106.Final.jar
./jars/cli/netty-transport-classes-epoll-4.1.106.Final.jar
./jars/cli/simpleclient_jetty-0.9.0.jar
./jars/cli/grpc-util-1.61.1.jar
./jars/cli/httpclient-4.5.3.jar
./jars/cli/failureaccess-1.0.1.jar
./jars/cli/netty-codec-mqtt-4.1.106.Final.jar
./jars/cli/commons-codec-1.9.jar
./jars/cli/shims-0.9.15.jar
./jars/cli/simpleclient_common-0.9.0.jar
./jars/cli/netty-codec-http-4.1.106.Final.jar
./jars/cli/checker-qual-3.37.0.jar
./jars/cli/netty-transport-rxtx-4.1.106.Final.jar
./jars/cli/jaxb-api-2.3.0.jar
./jars/cli/jetty-security-9.3.24.v20180605.jar
./jars/cli/netty-codec-4.1.106.Final.jar
./jars/cli/netty-transport-classes-kqueue-4.1.106.Final.jar
./jars/cli/netty-transport-native-unix-common-4.1.106.Final.jar
./jars/cli/error_prone_annotations-2.23.0.jar
./jars/cli/simpleclient-0.9.0.jar
./jars/cli/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
./jars/cli/protobuf-java-3.25.1.jar
./jars/cli/grpc-protobuf-lite-1.61.1.jar
./jars/cli/jakarta.inject-2.6.1.jar
./jars/cli/grpc-protobuf-1.61.1.jar
./jars/cli/rss-common-0.9.2.jar
./jars/cli/netty-resolver-4.1.106.Final.jar
./jars/cli/netty-resolver-dns-4.1.106.Final.jar
./jars/cli/simpleclient_pushgateway-0.9.0.jar
./jars/cli/simpleclient_httpserver-0.9.0.jar
./jars/cli/jetty-proxy-9.3.24.v20180605.jar
./jars/cli/jetty-http-9.3.24.v20180605.jar
./jars/cli/netty-transport-native-epoll-4.1.106.Final-linux-x86_64.jar
./jars/cli/grpc-core-1.61.1.jar
./jars/cli/log4j-api-2.23.0.jar
./jars/cli/netty-transport-sctp-4.1.106.Final.jar
./jars/cli/guava-32.1.3-jre.jar
./jars/cli/jakarta.annotation-api-1.3.5.jar
./jars/cli/jetty-client-9.3.24.v20180605.jar
./jars/cli/simpleclient_servlet-0.9.0.jar
./jars/cli/lz4-1.3.0.jar
./jars/cli/netty-codec-http2-4.1.106.Final.jar
./jars/cli/httpcore-4.4.6.jar
./jars/cli/netty-codec-memcache-4.1.106.Final.jar
./jars/cli/netty-transport-udt-4.1.106.Final.jar
./jars/cli/simpleclient_hotspot-0.9.0.jar
./jars/cli/jsr305-3.0.2.jar
./jars/cli/commons-lang3-3.10.jar
./jars/cli/jetty-servlet-9.3.24.v20180605.jar
./jars/cli/netty-resolver-dns-classes-macos-4.1.106.Final.jar
./jars/cli/netty-transport-native-kqueue-4.1.106.Final-osx-aarch_64.jar
./jars/cli/jakarta.validation-api-2.0.2.jar
./jars/cli/netty-resolver-dns-native-macos-4.1.106.Final-osx-aarch_64.jar
./jars/cli/grpc-netty-shaded-1.61.1.jar
./jars/cli/jackson-annotations-2.10.0.jar
./jars/cli/log4j-core-2.23.0.jar
./jars/cli/jackson-databind-2.10.0.jar
./jars/cli/protobuf-java-util-3.25.1.jar
./jars/cli/netty-transport-native-epoll-4.1.106.Final-linux-aarch_64.jar
./jars/cli/log4j-slf4j-impl-2.23.0.jar
./jars/dashboard/perfmark-api-0.26.0.jar
./jars/dashboard/annotations-4.1.1.4.jar
./jars/dashboard/grpc-stub-1.61.1.jar
./jars/dashboard/netty-codec-stomp-4.1.106.Final.jar
./jars/dashboard/netty-transport-native-kqueue-4.1.106.Final-osx-x86_64.jar
./jars/dashboard/netty-codec-smtp-4.1.106.Final.jar
./jars/dashboard/animal-sniffer-annotations-1.23.jar
./jars/dashboard/slf4j-api-1.7.36.jar
./jars/dashboard/jetty-server-9.3.24.v20180605.jar
./jars/dashboard/grpc-api-1.61.1.jar
./jars/dashboard/jetty-util-9.3.24.v20180605.jar
./jars/dashboard/rss-proto-0.9.2.jar
./jars/dashboard/RoaringBitmap-0.9.15.jar
./jars/dashboard/netty-all-4.1.106.Final.jar
./jars/dashboard/proto-google-common-protos-2.29.0.jar
./jars/dashboard/jetty-io-9.3.24.v20180605.jar
./jars/dashboard/javax.annotation-api-1.3.2.jar
./jars/dashboard/j2objc-annotations-2.8.jar
./jars/dashboard/picocli-4.5.2.jar
./jars/dashboard/netty-handler-4.1.106.Final.jar
./jars/dashboard/activation-1.1.1.jar
./jars/dashboard/grpc-context-1.61.1.jar
./jars/dashboard/netty-codec-dns-4.1.106.Final.jar
./jars/dashboard/netty-codec-redis-4.1.106.Final.jar
./jars/dashboard/jackson-core-2.10.0.jar
./jars/dashboard/javassist-3.25.0-GA.jar
./jars/dashboard/netty-transport-native-epoll-4.1.106.Final-linux-riscv64.jar
./jars/dashboard/netty-common-4.1.106.Final.jar
./jars/dashboard/netty-codec-haproxy-4.1.106.Final.jar
./jars/dashboard/gson-2.10.1.jar
./jars/dashboard/netty-buffer-4.1.106.Final.jar
./jars/dashboard/netty-resolver-dns-native-macos-4.1.106.Final-osx-x86_64.jar
./jars/dashboard/javax.servlet-api-3.1.0.jar
./jars/dashboard/netty-codec-xml-4.1.106.Final.jar
./jars/dashboard/hbase-shaded-jersey-4.1.4.jar
./jars/dashboard/netty-transport-4.1.106.Final.jar
./jars/dashboard/netty-handler-ssl-ocsp-4.1.106.Final.jar
./jars/dashboard/netty-handler-proxy-4.1.106.Final.jar
./jars/dashboard/netty-codec-socks-4.1.106.Final.jar
./jars/dashboard/netty-transport-classes-epoll-4.1.106.Final.jar
./jars/dashboard/simpleclient_jetty-0.9.0.jar
./jars/dashboard/grpc-util-1.61.1.jar
./jars/dashboard/failureaccess-1.0.1.jar
./jars/dashboard/netty-codec-mqtt-4.1.106.Final.jar
./jars/dashboard/shims-0.9.15.jar
./jars/dashboard/simpleclient_common-0.9.0.jar
./jars/dashboard/netty-codec-http-4.1.106.Final.jar
./jars/dashboard/checker-qual-3.37.0.jar
./jars/dashboard/netty-transport-rxtx-4.1.106.Final.jar
./jars/dashboard/jaxb-api-2.3.0.jar
./jars/dashboard/jetty-security-9.3.24.v20180605.jar
./jars/dashboard/netty-codec-4.1.106.Final.jar
./jars/dashboard/netty-transport-classes-kqueue-4.1.106.Final.jar
./jars/dashboard/netty-transport-native-unix-common-4.1.106.Final.jar
./jars/dashboard/error_prone_annotations-2.23.0.jar
./jars/dashboard/simpleclient-0.9.0.jar
./jars/dashboard/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
./jars/dashboard/protobuf-java-3.25.1.jar
./jars/dashboard/grpc-protobuf-lite-1.61.1.jar
./jars/dashboard/jakarta.inject-2.6.1.jar
./jars/dashboard/grpc-protobuf-1.61.1.jar
./jars/dashboard/rss-common-0.9.2.jar
./jars/dashboard/netty-resolver-4.1.106.Final.jar
./jars/dashboard/netty-resolver-dns-4.1.106.Final.jar
./jars/dashboard/simpleclient_pushgateway-0.9.0.jar
./jars/dashboard/simpleclient_httpserver-0.9.0.jar
./jars/dashboard/jetty-proxy-9.3.24.v20180605.jar
./jars/dashboard/jetty-http-9.3.24.v20180605.jar
./jars/dashboard/netty-transport-native-epoll-4.1.106.Final-linux-x86_64.jar
./jars/dashboard/grpc-core-1.61.1.jar
./jars/dashboard/log4j-api-2.23.0.jar
./jars/dashboard/netty-transport-sctp-4.1.106.Final.jar
./jars/dashboard/guava-32.1.3-jre.jar
./jars/dashboard/jakarta.annotation-api-1.3.5.jar
./jars/dashboard/jetty-client-9.3.24.v20180605.jar
./jars/dashboard/simpleclient_servlet-0.9.0.jar
./jars/dashboard/lz4-1.3.0.jar
./jars/dashboard/netty-codec-http2-4.1.106.Final.jar
./jars/dashboard/dashboard-0.9.2.jar
./jars/dashboard/netty-codec-memcache-4.1.106.Final.jar
./jars/dashboard/netty-transport-udt-4.1.106.Final.jar
./jars/dashboard/simpleclient_hotspot-0.9.0.jar
./jars/dashboard/jsr305-3.0.2.jar
./jars/dashboard/commons-lang3-3.10.jar
./jars/dashboard/jetty-servlet-9.3.24.v20180605.jar
./jars/dashboard/netty-resolver-dns-classes-macos-4.1.106.Final.jar
./jars/dashboard/netty-transport-native-kqueue-4.1.106.Final-osx-aarch_64.jar
./jars/dashboard/jakarta.validation-api-2.0.2.jar
./jars/dashboard/netty-resolver-dns-native-macos-4.1.106.Final-osx-aarch_64.jar
./jars/dashboard/grpc-netty-shaded-1.61.1.jar
./jars/dashboard/jackson-annotations-2.10.0.jar
./jars/dashboard/log4j-core-2.23.0.jar
./jars/dashboard/jackson-databind-2.10.0.jar
./jars/dashboard/protobuf-java-util-3.25.1.jar
./jars/dashboard/netty-transport-native-epoll-4.1.106.Final-linux-aarch_64.jar
./jars/dashboard/log4j-slf4j-impl-2.23.0.jar
./jars/client/mr/rss-client-mr-0.9.2-shaded.jar
./jars/client/tez/rss-client-tez-0.9.2-shaded.jar
./jars/client/spark3/rss-client-spark3-shaded-0.9.2.jar
./jars/client/spark2/rss-client-spark2-shaded-0.9.2.jar
```

### Affects Version(s)

master & 0.9.2-rc1

### Uniffle Server Log Output

_No response_

### Uniffle Engine Log Output

_No response_

### Uniffle Server Configurations

_No response_

### Uniffle Engine Configurations

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?

Add left jars with Apache license

### Why are the changes needed?
Fix #2307

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
No need.

### CODE REVIEW

**Review by kaijchen — APPROVED** [review]
LGTM

**Comment by pjfanning** [comment]
I don't think org.javassist:javassist is Apache licensed.

https://mvnrepository.com/artifact/javassist/javassist says it is LGPL and MPL licensed. LGPL is a problem but MPL is Category B and is ok. So we can use the jar.

**Comment by pjfanning** [comment]
> I don't think org.javassist:javassist is Apache licensed.
> 
> https://mvnrepository.com/artifact/javassist/javassist says it is LGPL and MPL licensed. LGPL is a problem but MPL is Category B and is ok. So we can use the jar.

It actually looks like org.javassist:javassist is licensed under 3 licenses (the 2 above and Apache license too which is better).

https://repo1.maven.org/maven2/org/javassist/javassist/3.30.2-GA/javassist-3.30.2-GA.pom

Could we update the LICENSE-binary to name the 3 licenses instead of grouping it under Apache license alone?

**Comment by jerqi** [comment]
> > I don't think org.javassist:javassist is Apache licensed.
> > https://mvnrepository.com/artifact/javassist/javassist says it is LGPL and MPL licensed. LGPL is a problem but MPL is Category B and is ok. So we can use the jar.
> 
> It actually looks like org.javassist:javassist is licensed under 3 licenses (the 2 above and Apache license too which is better).
> 
> https://repo1.maven.org/maven2/org/javassist/javassist/3.30.2-GA/javassist-3.30.2-GA.pom
> 
> Could we update the LICENSE-binary to name the 3 licenses instead of grouping it under Apache license alone?

OK.

**Comment by jerqi** [comment]
> LGPL

Added.

**Comment by jerqi** [comment]
@pjfanning Do you have other suggestion?

**Comment by jerqi** [comment]
@pjfanning Do you have any other suggestion?

**Comment by pjfanning** [comment]
> @pjfanning Do you have any other suggestion?

seems ok to me

**Review by zhengchenyu — APPROVED** [review]

File: LICENSE-binary [hunk 1]
```diff
@@ -303,3 +344,8 @@ org.apache.hbase.thirdparty:hbase-shaded-jersey
 The ISC License
 ---------------
 rimraf
+
+The Apache Software License, Version 2.0, LGPL License or MPL License
```
Comment (justinmclean) [inline LICENSE-binary:348]: When something is available under multiple licenses, you should select the license that is compatable with the ASF license/policy. See https://www.apache.org/legal/resolved.html#mutually-exclusive

**Pull request merged**